### PR TITLE
Add bulgarian and pseudo language

### PR DESCRIPTION
**Pull request by daniserejo**
Add bulgarian and pseudo language.

https://bulgarian--storecomponents.myvtex.com/admin/apps/vtex.reviews-and-ratings@2.10.2/setup/

### CODE REVIEW

**Review by wender — CHANGES_REQUESTED** [review]
Missing CHANGELOG file

**Review by camilavcoutinho — COMMENTED** [review]
@daniserejo you need to delete ach-UG.json and pt-BR.json

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added 
+
+- I18n bs-BA
```
Comment (camilavcoutinho) [inline CHANGELOG.md:11]:
```suggestion
- I18n pseudo language to implement In Context tool.
```

**Review by camilavcoutinho — CHANGES_REQUESTED** [review]

**Comment by camilavcoutinho** [comment]
@daniserejo could you rebase? there is a conflict in the changelog

**Comment by camilavcoutinho** [comment]
@daniserejo this PR needs rebase and the deletion of pt-BR.json, please 😊

**Review by wender — APPROVED** [review]

**Review by igorbrasileiro — APPROVED** [review]

**Review by camilavcoutinho — APPROVED** [review]

**Pull request merged**